programming-fundamentals/core/data/data-types.md

```diff
@@ -0,0 +1,47 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Data Types
+
+---
+## Content
+
+In real-life you can split items into different types. All items of that type have some common characteristics. For example, you could think of a *leaf* as being a type. Although not all leafs look the same, they still have many characteristics in common.
+
+Just as in real-life, everything in programming has a data type. The most common ones are:
+- numerical
+- text-based
+- lists
+
+There are many more types, but you'll start learning them as you dive into a specific programming language. This is because each language might have different implementations of complex data types which can't be generalised, or they are beyond the scope of this course.
+
+Data types are very important! They decide what actions you can take on that piece of data. One good example is that you can't add the number `1` to the text `"one"`.
+
+---
+## Practice
+
+The most common data types are:
+- numerical
+- text-based
+- lists
```

Comment:
this is not a mistake, I'm trying a new type of "question" which is actually a reminder of the most important concept in the previous insight